res/CircleSchema/0.8/circle_schema.fbs

```diff
@@ -831,6 +837,7 @@ table Conv2DOptions {
   dilation_h_factor:int = 1;
   // Parameters for Conv2D version 8 or above.
   // When set, quantized_bias_type defines the dtype for both bias and accumulator.
+  weight_compression_type:WeightCompressionType = NONE;
```

Comment:
we need to upgrade to 0.9 and this requires lots of other changes.

CC @hseok-oh